src/enterprise/enterprise.proto

```diff
@@ -7,16 +7,32 @@ import "google/protobuf/timestamp.proto";
 
 // Enterprise data structures
 
-// EnterpriseRecord is the record we store in etcd for a Pachyderm enterprise
-// token that has been provided to a Pachyderm cluster
-message EnterpriseRecord {
+// LicenseRecord is the record we store in etcd for a Pachyderm enterprise
+// token that has been provided to a Pachyderm license server
+message LicenseRecord {
```

Comment:
`LicenseRecord` contains an activation code and an optional, user-supplied expiry. This is reused in the license server and the enterprise server so we can test expiration in both places without generating new licenses.